docs/src/main/sphinx/functions/geospatial.md

```diff
@@ -86,7 +86,7 @@ Array elements must not be `NULL` or empty.
 The returned geometry may not be simple and may contain duplicate points if input array has duplicates.
 :::
 
-:::{function} ST_Point(double, double) -> Point
+:::{function} ST_Point(lat: double, lon: double) -> Point
```

Comment:
This is a change that is necessary since the Python function support does not allow identical parameters. I am fine with using this .. maybe as a follow up we should consider doing that for all functions? Wdyt @martint and @electrum

Reply:
I like this as I'm guessing this makes the rendered documentation easier to read. Do you have a screenshot?

Reply:
The current docs look like

<img width="623" alt="Screenshot 2024-04-23 at 4 41 35 PM" src="https://github.com/trinodb/trino/assets/9230/27d97870-8529-4e60-a7f0-b2c8cfc83c89">

Reply:
@electrum correct, but now the python function thingy, doesn't allow to have the same argument twice in the argument list without labels (which makes sense actually if you consider this to be a python function, not a SQL one).

Reply:
<img width="804" alt="image" src="https://github.com/trinodb/trino/assets/145658/da9a63fe-5f6b-4b3f-b1cd-3dbc15fe54b4">